package.json

```diff
@@ -52,7 +52,7 @@
   },
   "scripts": {
     "husky:install": "husky install",
-    "test": "jest",
+    "test": "jest -w 4",
```

Comment:
My machine runs so many tests in parallel that it actually gets a lot slower doing so because it runs out of memory. In a steady state with everything cached (after a few runs) 4 is also the sweet spot where they run quickest.

I will change the option to the long version for readability.